### ISSUE TITLE
Needs permanent icon

### ISSUE THREAD

**Issue by malcb** [perfectapi/CKP#58]
I find CKP works fine on chromebook for simple sites with just user/pass, so 5 stars for that.  It would be nice if it could support keepass's password pick but I expect that is asking too much.

 However, where it falls down is a typical banking login, page 1 would be user/pass where CKP icon appears in the URL bar and CKP works fine.  Page 2 of the banking login would be a dropdown boxes, now I want to access CKP again and view the entry to remind me of the data for the dropdown but CKP icon is no longer in the url bar (presumably because this page only has dropdown boxes, not user/pass fields).

What CKP needs then is either a way to force it to appear such as a list of sites it should always appear on, or an icon always shown that can be clicked to open the database.  I think the former is a nicer solution.  Could it be done by matching the domain part of the saved URL for autofill?  That would typically be the bank first login page.  CKP would then always be available when on the bank website which would cover most if no all needs.

BTW my current work around for this is to open in another tab a user/pass web page to get CKP to appear.  Then I can manually search the database for the right entry and view it before switching back to the tab I'm really using.

**Comment by agrueneberg** [comment]
Seconded, it would be nice to be able to open the safe when no login boxes are present. I'm also navigating to certain pages that I know have login boxes just to retrieve some extra information such as PINs or answers to security questions.

**Comment by tmajibon** [comment]
I'm not 100% sure, but I think Google resolved this one with a recent Chrome update.

I opened my browser the other day and it even had one of those help pop-ups talking about it.

It seems all the extension icons are going to be permanently in the top right (beside the hamburger menu). They give you an option to hide them (if you're trying to figure out how to unhide, it actually just moves it to the end of the list and resizes the space so that it's hidden, you can drag/drop to resize the bar to get it back.

**Comment by DukeyToo** [comment]
Yeah, they're changing the way the icons display, but it does not solve the issue - the icon will still be disabled in this scenario.

**Comment by tmajibon** [comment]
Ah, I see.

Most of the sites I looked at triggered it regardless of a prompt appearing or not. But I found one where it was disabled. So apparently some sites trick it out in some ways (like Github itself is showing even though a password prompt isn't present on the loaded page).
